src/Login.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react'
+import {
+  Grid,
+  Row,
+  Col,
+  FormGroup,
+  ControlLabel,
+  FormControl,
+  HelpBlock,
+  Button,
+  Alert
+} from 'react-bootstrap'
+import { withRouter } from 'react-router'
+
+import NavBar from './components/NavBar'
+import { Api } from './utils'
+
+function FieldGroup({ id, label, help, ...props }) {
```

Comment:
We already have a generalized `FieldGroup` component, and it looks like the `help` prop isn't being used. We should be able to use the generalized component